edx_when/utils.py

```diff
@@ -10,13 +11,34 @@
     Schedule = None
 
 
-def get_schedule_for_user(user_id, course_key):
+def get_schedule_for_user(user_id, course_key, use_cached=True):
     """
     Return the schedule for the user in the course or None if it does not exist or the Schedule model is undefined.
     """
-    if Schedule:
-        try:
-            return Schedule.objects.get(enrollment__user__id=user_id, enrollment__course__id=course_key)
-        except ObjectDoesNotExist:
-            pass
-    return None
+    # If Schedule is not defined, there's nothing to query, so return None. This
+    # hackiness is happening because the Schedule model is in edx-platform at
+    # the moment.
+    if not Schedule:
+        return None
```

Comment:
This is not covered by any tests at this moment, because it never happens in practice–either we're running inside the edx-platform process and the import succeeds, or we're in edx-when tests and it's been mocked with a `DummySchedule` object.

@ilee2u: I'm inclined to just ignore this omission (no test existed for the similar check it replaced), but I can add a test for this if you think it's warranted.